vmck/migrations/0001_initial.py

```diff
@@ -15,21 +15,27 @@ class Migration(migrations.Migration):
         migrations.CreateModel(
             name='Artifact',
             fields=[
-                ('id', models.AutoField(auto_created=True, primary_key=True, serialize=False, verbose_name='ID')),
+                ('id', models.AutoField(auto_created=True, primary_key=True,
```

Comment:
Please set up [one of these](https://github.com/hoover/snoop2/blob/d539509681b2ad01a100ed4d2c64cd0c7ece513b/.flake8#L7) and ignore the line length for migrations, so future migrations won't crash flake8.